bridge/client/app/dashboard/dashboard.component.ts

```diff
@@ -15,10 +15,10 @@ export class DashboardComponent implements OnInit, OnDestroy{
   public logoInvertedUrl = environment?.config?.logoInvertedUrl;
   public isQualityGatesOnly: boolean;
 
-  private readonly _projectTimerInterval = 30 * 1000;
+  private readonly _projectTimerInterval = 5 * 1000;
```

Comment:
Is this on purpose? I think 30 seconds was already okay, wasn't it?

Reply:
Yes you are right, this was a leftover from testing